File: 30 Days of Code/Day 1/Data Types.go
package main

import (
  "fmt"
  "os"
  "bufio"
  "strconv"
)

func main() {
  var _ = strconv.Itoa // Ignore this comment. You can still use the package "strconv".
  var i uint64 = 4
  var d float64 = 4.0
  var s string = "HackerRank "

  scanner := bufio.NewScanner(os.Stdin)
  // Print the sum of both integer variables on a new line.
  var x uint64; var y float64; fmt.Scanf("%d\n%f\n", &x ,&y)
  // Print the sum of the double variables on a new line.
  fmt.Println(i + x); fmt.Printf("%.1f\n", d + y); scanner.Scan()
  // Concatenate and print the String variables on a new line
  var z string = scanner.Text(); fmt.Print(s + z, "\n")
}

File: 30 Days of Code/Day 1/Data Types.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
    int i = 4;
    double d = 4.0;
    char s[] = "HackerRank ";
    // Print the sum of both integer variables on a new line.
    int x; scanf("%d", &x); printf("%d\n", x + i);
    // Print the sum of the double variables on a new line.
    double y; scanf("%lf", &y); printf("%.1f\n", d + y);
    // Concatenate and print the String variables on a new line
    char z[50]; printf(s); while(scanf("%s",z)>0) printf("%s ", z);
    return 0;
}
